shopelectro/views/ecommerce.py

```diff
@@ -97,10 +96,10 @@ def post(self, request):
         # Took form fields from Yandex docs https://goo.gl/afKfsz
         response_data = {
             'yandex_kassa_link': settings.YANDEX_KASSA_LINK, # Required
-            'shopId': SHOP['id'], # Required
-            'scid': SHOP['scid'], # Required
-            'shopSuccessURL': SHOP['success_url'],
-            'shopFailURL': SHOP['fail_url'],
+            'shopId': settings.SHOP['id'],  # Required
```

Comment:
@YozhEzhi , pep8 requires two whitespaces. I fix it every time, when i notice it.
https://www.python.org/dev/peps/pep-0008/#inline-comments